libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRoute.kt

```diff
@@ -48,8 +49,24 @@ class NavigationRoute internal constructor(
     val routeIndex: Int,
     val routeOptions: RouteOptions,
     internal val nativeRoute: RouteInterface,
+    internal val expectedClosures: List<List<Closure>>,
```

Comment:
nit: this should be a property of the class.

Reply:
But it is a property.